src/applications/vaos/tests/appointment-list/components/PastAppointmentsListV2.unit.spec.jsx

```diff
@@ -128,15 +138,20 @@ describe('VAOS <PastAppointmentsListV2>', () => {
 
   it('should show information with facility name', async () => {
     const pastDate = moment().subtract(3, 'days');
-    const appointment = getVAAppointmentMock();
-    appointment.attributes = {
-      ...appointment.attributes,
-      startDate: pastDate.format(),
-      clinicFriendlyName: 'C&P BEV AUDIO FTC1',
-      facilityId: '983',
-      sta6aid: '983GC',
+    const data = {
+      id: '1234',
+      currentStatus: 'CHECKED OUT',
+      kind: 'clinic',
+      clinic: 'fake',
+      start: pastDate.format(),
+      locationId: '983GC',
+      status: 'booked',
```

Comment:
This should probably be a status of fulfilled, since that's what most past appointments will be (especially if they have a vista status of checked out)

Reply:
Done.